### PR TITLE
Remove sftp

### PR DESCRIPTION
**Pull request by kolmanv**

### CODE REVIEW

File: examples/tcp_example.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# This example shows bi-directional simple communication between TCP server and client.
+
+require 'log'
+require 'networking/tcp'
+
+BBFS::Params.log_write_to_console = 'true'
+BBFS::Params.log_debug_level = 0
+BBFS::Params.log_param_number_of_mega_bytes_stored_before_flush = 0
+BBFS::Params.log_param_max_elapsed_time_in_seconds_from_last_flush = 0
```
Comment (yarondbb) [inline examples/tcp_example.rb:9]: remove lines 7-9 these are the defaults

File: examples/tcp_example.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# This example shows bi-directional simple communication between TCP server and client.
+
```
Comment (yarondbb) [inline examples/tcp_example.rb:2]: add execution examples:
        \bbfs>ruby -Ilib examples\tcp_example.rb  # to run with default parameters
        \bbfs>ruby -Ilib examples\tcp_example.rb --log_debug_level=3  # to see more log details

File: examples/tcp_example.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+require 'log'
+require 'networking/tcp'
+
+BBFS::Params.log_write_to_console = 'true'
```
Comment (yarondbb) [inline examples/tcp_example.rb:6]: replace with:
BBFS::Params['log_write_to_console'] = true

File: examples/tcp_example.rb [hunk 4]
```diff
@@ -0,0 +1,38 @@
+# This example shows bi-directional simple communication between TCP server and client.
+
+require 'log'
```
Comment (yarondbb) [inline examples/tcp_example.rb:3]: add:
require 'params'

File: examples/tcp_example.rb [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+require 'networking/tcp'
+
```
Comment (yarondbb) [inline examples/tcp_example.rb:5]: add:
BBFS::Params.init ARGV

File: lib/networking/tcp.rb [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+        return Thread.new do
+          loop do
+            Log.debug3('Start blocking read (TCPClient).')
+            # Blocking read.
+            open_socket unless socket_good?
+            if !socket_good?
+              Log.warning('Socket not good, breaking client reading thread.')
+              break
+            end
+            Log.debug1("socket:#{@tcp_socket.string}")
```
Comment (yarondbb) [inline lib/networking/tcp.rb:145]: @tcp_socket.string raises excpetion: (execution plaform is win7)
    undefined method `string' for #<TCPSocket:fd 5> (NoMethodError)

File: examples/tcp_example.rb (hunk 1 above)
Comment (kolmanv) [inline examples/tcp_example.rb:9]: Done.

File: examples/tcp_example.rb (hunk 2 above)
Comment (kolmanv) [inline examples/tcp_example.rb:2]: Done.

File: examples/tcp_example.rb (hunk 3 above)
Comment (kolmanv) [inline examples/tcp_example.rb:6]: Done.

File: examples/tcp_example.rb (hunk 4 above)
Comment (kolmanv) [inline examples/tcp_example.rb:3]: Done.

File: examples/tcp_example.rb (hunk 5 above)
Comment (kolmanv) [inline examples/tcp_example.rb:5]: Done.

File: lib/networking/tcp.rb (hunk 6 above)
Comment (kolmanv) [inline lib/networking/tcp.rb:145]: Done.

**Comment by kolmanv** [comment]
I have merged this pull request by mistake. Sorry.

**Pull request merged**